### PR TITLE
add addValues method in EnumListType

### PR DESCRIPTION
**Pull request by Alban-io**

### CODE REVIEW

**Comment by Alban-io** [comment]
Because I don't want to exclude Iterator Object ;)

File: core/lib/Thelia/Type/EnumListType.php [hunk 1]
```diff
@@ -36,6 +36,17 @@ public function addValue($value)
         }
     }
 
+    /**
+     * @param array $values
+     * @since 2.3.0
+     */
+    public function addValues($values = array())
```
Comment (lunika) [inline core/lib/Thelia/Type/EnumListType.php]: IMO if you don't want to use type hinting here, you have to test if `$values` is an `array` or an instance of `Traversable` and throw an exception if it's not the case

File: core/lib/Thelia/Type/EnumListType.php [hunk 2]
```diff
@@ -36,6 +36,17 @@ public function addValue($value)
         }
     }
 
+    /**
+     * @param array $values
```
Comment (lunika) [inline core/lib/Thelia/Type/EnumListType.php]: This is not really true with your implementation

**Comment by Alban-io** [comment]
I agree with your opinion @lunika, i will fix it.

**Comment by gillesbourgeat** [comment]
Thanks !

**Pull request merged**